package.json

```diff
@@ -44,7 +44,7 @@
     "rollup-plugin-node-resolve": "^3.0.2",
     "rollup-plugin-sass": "^0.5.3",
     "rollup-plugin-uglify": "^2.0.1",
-    "serve": "^6.4.8"
+    "serve": "^6.4.8 <6.5.4"
```

Comment:
dat.gui's build system is not currently compatible with node 8 or above due to the version of node-sass used by rollup so I built it with node 7. However "serve" lost compatibility with node 7 in 6.5.4 so clamped it back down.

Reply:
Interesting... that's `rollup-plugin-sass` I suppose with the old version I suppose?

Reply:
Yeah, although I'm sure there are newer versions that are compatible for building with node 8 it's just a matter of doing the necessary upgrades to package json and / or the loockfile and then verifying.